### PR TITLE
[berkeley] Make noisy logs during bootstrap quieter

### PR DESCRIPTION
**Pull request by mrmr1993**
This PR removes the (extremely verbose) JSON diff for snark work and txn pool diffs that time-out. The diff is still included in the log files, but not included in the message on the command line.

In particular, this removes a large source of noise in the log messages emitted during bootstrap and catchup, where these diffs will regularly time out.

### CODE REVIEW

**Comment by mrmr1993** [comment]
!ci-build-me

**Review by georgeee — APPROVED** [review]

File: src/lib/network_pool/pool_sink.ml [hunk 1]
```diff
@@ -158,7 +158,7 @@ module Base
               | None ->
                   let diff = Envelope.Incoming.data env' in
                   [%log error]
```
Comment (emberian) [inline src/lib/network_pool/pool_sink.ml:160]:
```suggestion
                  [%log trace]
```

maybe, if it's not that big a deal?

Comment (mrmr1993) [reply src/lib/network_pool/pool_sink.ml:160]: Maybe, but this seemed easier than making that concrete decision.

**Comment by mrmr1993** [comment]
!ci-build-me

**Comment by deepthiskumar** [comment]
!approved-for-mainnet

**Pull request merged**